tools/build.py

```diff
@@ -9,6 +9,17 @@
 PREFIX = "zen_"
 ##########################
 
+def tryHemttBuild(projectpath):
+    hemttExe = os.path.join(projectpath, "hemtt.exe")
```

Comment:
What if I would want to build it on Linux too?

Reply:
I took this code from the CBA repo, I don't have enough experience with Linux and its file system, or with python, to know how to make it work.

Reply:
You shouldn't use `build.py`, but use `hemtt` binary directly.

Reply:
Yep, that's true. I'd guess I'm just overthinking it too much.